xmodule/capa/safe_exec/safe_exec.py

```diff
@@ -373,25 +373,33 @@ def report_arm(arm, globals_dict, emsg, unexpected_exception):
             set_custom_attribute(f'codejail.darklaunch.status.{arm}', 'ok' if emsg is None else 'safe_error')
             set_custom_attribute(f'codejail.darklaunch.exception.{arm}', None)
 
-        # Logs include full globals and emsg
-        log.info(
-            f"Codejail darklaunch {arm} results for slug={slug}: globals={globals_dict!r}, "
-            f"emsg={emsg!r}, exception={unexpected_exception!r}"
-        )
-
-    report_arm('local', globals_local, emsg_local, unexpected_exc_local)
-    report_arm('remote', globals_remote, emsg_remote, unexpected_exc_remote)
+    report_arm('local', emsg_local, unexpected_exc_local)
+    report_arm('remote', emsg_remote, unexpected_exc_remote)
 
     # If the arms can't be compared (unexpected errors), stop early -- the rest
     # is about output comparison.
     if not can_compare_output:
         set_custom_attribute('codejail.darklaunch.globals_match', 'N/A')
         set_custom_attribute('codejail.darklaunch.emsg_match', 'N/A')
+        log.info(
+            "Codejail darklaunch had unexpected exception for "
+            f"course={limit_overrides_context}, slug={slug}:\n"
```

Comment:
I didn’t see a test that uses this like a course.

Reply:
Yeah, none of the tests have had a sample value for context or slug. Do you think it's necessary? I could either change it for all of them or add more parameters to the `run_dark_launch` function so that we can exercise both string/None.

Reply:
Exercising both would be nice, but if just one, I think I'd prefer them all to change to String. Thoughts?

Reply:
Sure, I've pushed up a change to set both context and slug for all of them.